.github/workflows/e2e.ci.yaml

```diff
@@ -43,98 +45,41 @@ jobs:
           repository: apache/skywalking
           submodules: true
           path: main
-          ref: 6228d450e1b0792dd9875ad68984c89c81ece742
+          ref: 8436135dc919351a6d0ab42ff48c85e1b3104b53
+      - uses: actions/checkout@v2
+        with:
+          repository: apache/skywalking-satellite
+          submodules: true
+          path: satellite
+          ref: 4b97fd255b6459fe9d3fc4f88c0d8f682fc2a364
 
       - uses: actions/cache@v2
         with:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-maven-
 
-      - name: Build Docker Image
+      - name: Build SkyWalking OAP Docker Image
```

Comment:
Replace this step with the snapshot docker images from main repo, no need to build here